editor.planx.uk/src/@planx/components/FindProperty/Public/Autocomplete.tsx

```diff
@@ -184,16 +191,21 @@ export default function PickOSAddress(props: PickOSAddressProps): FCReturn {
         />
       </InputLabel>
       {sanitizedPostcode && (
-        /* @ts-ignore */
-        <address-autocomplete
-          id="address-autocomplete"
-          data-testid="address-autocomplete-web-component"
-          postcode={sanitizedPostcode}
-          initialAddress={selectedOption?.title || ""}
-          osProxyEndpoint={`${process.env.REACT_APP_API_URL}/proxy/ordnance-survey`}
-          arrowStyle="light"
-          labelStyle="static"
-        />
+        <ErrorWrapper
+          error={props.addressAutocompleteError}
+          id={"address-autocomplete"}
+        >
+          {/* @ts-ignore */}
+          <address-autocomplete
+            id="address-autocomplete"
+            data-testid="address-autocomplete-web-component"
+            postcode={sanitizedPostcode}
+            initialAddress={selectedOption?.title || ""}
+            osProxyEndpoint={`${process.env.REACT_APP_API_URL}/proxy/ordnance-survey`}
+            arrowStyle="light"
+            labelStyle="static"
+          />
+        </ErrorWrapper>
```

Comment:
Wrapping the web component in our error wrapper isn't perfect because it means the error message & select label will be out of normal order, with the error message _above_ the label rather than _below_.

We can: 
1. Ignore this for now - it'll never be visible at the same time as the postcode error message, so hard to notice
2. Implement an error wrapper directly in our web component code (and create/pass new `error` prop?)
    - Looking through all the alpha gov examples of this component, none of them have error handling :melting_face: https://alphagov.github.io/accessible-autocomplete/examples/
3. Other suggestions?

![Screenshot from 2024-05-13 20-36-07](https://github.com/theopensystemslab/planx-new/assets/5132349/14a47d61-f314-47bc-9a73-8702c64ccac4)